### PR TITLE
[4.0] fix media edit modal cancel button

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue #23832 .

### Summary of Changes
The cancel button in the edit view wasn't closing the modal, now it does

@bahl24, @Chaosxmk please test this

### Documentation Changes Required

### CODE REVIEW

**Comment by ghazal** [comment]
I have tested this item :white_check_mark: successfully on 3d0d0792aa0dc4f13d3c3860aa6f092147b43a3d<br /><br />After modifying /build/media_source/com_media/js/edit-images.es6.js
and running
npm run build:js

looks like it works, ie it closes the modal and falls back to the content tab.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23880">issues.joomla.org/tracker/joomla-cms/23880</a>.</sub>

**Comment by bahl24** [comment]
@dgrammatiko On clicking cancel, the whole window closes rather than that operation(like edit in this case). We expect-
screenshot from 2019-02-11 23-52-26
Actual- Window closes

**Comment by dgrammatiko** [comment]
@bahl24 my misunderstanding then what cancel should been be doing in this case. Anyways it should now return to list view

**Comment by bahl24** [comment]
@dgrammatiko I think it's fine now.

File: build/media_source/com_media/js/edit-images.es6.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ Joomla.MediaManager = Joomla.MediaManager || {};
     extension: options.uploadPath.split('.').pop(),
     contents: `data:image/${options.uploadPath.split('.').pop()};base64,${options.contents}`,
   };
+
```
Comment (bahl24) [inline build/media_source/com_media/js/edit-images.es6.js]: Is this new line intentional?

Comment (dgrammatiko) [reply build/media_source/com_media/js/edit-images.es6.js]: I think so

Comment (bahl24) [reply build/media_source/com_media/js/edit-images.es6.js]: I think you added the new line, by mistake.

**Review by bahl24 — APPROVED** [review]

**Comment by anuragteapot** [comment]
@bahl24 You need to test this PR. Use Joomla issue tracker.

**Comment by anuragteapot** [comment]
@bahl24 No, need to confuse. Go here https://issues.joomla.org/

And mark your test.

**Comment by bahl24** [comment]
I have tested this item :white_check_mark: successfully on 6c358b1207a153aa13622e0e6b8573b403658bad<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23880">issues.joomla.org/tracker/joomla-cms/23880</a>.</sub>

**Comment by bahl24** [comment]
> @bahl24 No, need to confuse. Go here https://issues.joomla.org/
> 
> And mark your test.

@Anu1601CS Thanks for your help.

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 6c358b1207a153aa13622e0e6b8573b403658bad<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23880">issues.joomla.org/tracker/joomla-cms/23880</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23880">issues.joomla.org/tracker/joomla-cms/23880</a>.</sub>

**Comment by dgrammatiko** [comment]
Thanks

**Pull request merged**